src/Components/Resource/ResourceDetailsUpdate.tsx

```diff
@@ -159,7 +167,7 @@ export const ResourceDetailsUpdate = (props: resourceProps) => {
   const handleSubmit = async (e: any) => {
     const validForm = validateForm();
 
-    if (validForm) {
+    if (!validForm) {
```

Comment:
@patelaryan7751 went through the code, it seems we are returning weather the form is invalid or not, the variable names are not correct and confusing, so can you please change the variable name to invalidForm or can you negate the return in validateForm.

I think second option would be better, do change which works for you.